Victoria - bp557/victoria_week7.md

```diff
@@ -0,0 +1,7 @@
+My survey is designed to help golf clubs better do their service and marketing.
+
+This is the updated version of the survey.
+
+Link to survey on Google Drive (revised version): https://docs.google.com/spreadsheets/d/1EUVXyivrmnCy4r5_S9_nshZwb4G1xBzVfRY_hEFMorQ/edit?usp=sharing
+
+Link to SurveyCTO (revised form): https://gui2de.surveycto.com/forms/victoria_week6/designer.html?view=test&caseid=
```

Comment:
Hi! Here's my feedback on your survey. I number my points for clarity and reference (they do not refer to question numbers).

1. Your phone number question allows non-numbers. I'd recommend putting a constraint on the format to XXX-XXX-XXXX (with some wiggle rooms about dashes vs. dots vs. spaces vs. nothing).

2. Your email question also allows any input. Perhaps limit to XXX@XXX.com/org/edu/net or something of that nature.

3. You might consider making the age and birthday questions optimal, as some survey takers may not want to disclose such information, but you don't want false information or less data collection.

4. Unless I missed something, the birthday question requires clicking back every single month, starting from today's date. Allowing the user to type the date directly (e.g., see the template from class) and/or start from a year closer to your target demographics' mean birthday would be ideal. Also, you may want to constrain the age years (e.g., no 2-year-olds are filling out this survey).

5. The continent question is a bit confusing. Do you mean the continent I grew up on? I live in now? My favorite? Also, depending on what you mean, maybe exclude Antarctica. Same advice about making this an optional question.

6. When I select "no" on the family members question, it still asks me how many family members I have. You should condition these questions on the response to that question. Also, what do you mean by family member? Do cousins, aunts, and uncles count? What about in-laws? There is no constraint on this question--e.g., I can enter 1,000,000 as my sibling count.

7. In the family member 1 question, there is a typo--it should read "family _member_ 1" not "family number 1." Also, consider constraining this question to letters, I can currently enter numbers. Also, I am asked this question even if I input 0 for my number of family members in the previous question.

8. Consider constraining the family member age question to numbers only.

9. Consider making question on past week's activities optional. Also, if I select no for all activities, it still asks me on the subsequent page how many hours I spent doing my activities.

10. I think you should vastly simplify your 168-hour check. Two sentences at the max.

11. I was asked extensive questions about golf despite selecting no for "have you played golf in the past week" question.

12. Does the income question refer to my survey respondent's personal income or his/her household income?

Let me know if you have any questions/comments/concerns about my feedback above. Overall, nice job!

Reply:
Hi F, thanks for your detailed comments, I have made some changes to my survey, pls feel free to have a look at it. Here's the new link to my google sheet: https://docs.google.com/spreadsheets/d/1l2M0u6W4tQDCkk-BznCmtgRzbQjIZl9u9jxyUeno4uQ/edit?usp=sharing